traits/tests/test_map.py

```diff
@@ -64,5 +64,4 @@ class Person(HasTraits):
         self.assertEqual(reconstituted.validate(p, "married", "yes"), "yes")
 
         with self.assertRaises(TraitError):
-            self.assertEqual(reconstituted.validate(p, "married", "uknown"),
-                             "unknown")
+            reconstituted.validate(p, "married", "uknown")
```

Comment:
OCD, I know, but can we change `"uknown"` to `"unknown"`?

Reply:
Lalalalalala...  🐛  okay :)